pdc/__init__.py

```diff
@@ -8,9 +8,11 @@
 
 VERSION = "0.5.0-0.1.beta"
```

Comment:
:+1: 
how about update this version to current upstream release version "v0.1.0-alpha"?